### PR TITLE
Mntor-3133 - Verify that the High risk data breaches step is displayed correctly

### PR DESCRIPTION
**Pull request by mozilloid**
# References:

Jira: MNTOR-3133
Figma:

# Description
Simple test that checks the existence of "High Risk data breaches" section when navigated to it. Also, logs in as 'exposures_started' account to get the user that meets the preconditions criteria.

# Screenshot (if applicable)

Not applicable.

# How to test
npx playwright test -g "Verify that the High risk data breaches step is displayed correctly"

# Checklist (Definition of Done)

- [ ] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/e2e/specs/dashboard.spec.ts [hunk 1]
```diff
@@ -127,6 +133,94 @@
...
+    await openProfileMenuItem(
+      dashboardPage.profileSettings,
+      /.*\/user\/settings.*/,
+    );
+
+    const baseUrl = process.env["E2E_TEST_BASE_URL"];
+    expect(baseUrl).toBeTruthy();
+    await openProfileMenuItem(
+      dashboardPage.profileSignOut,
+      new RegExp(baseUrl!),
```
[the remark this answers is not in the document]
Comment (mozilloid) [reply src/e2e/specs/dashboard.spec.ts:221]: There are just tests, shouldn't matter

File: src/e2e/pages/authPage.ts [hunk 2]
```diff
@@ -39,16 +41,16 @@ export class AuthPage {
...
-    );
+  async enterPassword(optionalPassword?: string) {
+    const password =
+      optionalPassword || (process.env.E2E_TEST_ACCOUNT_PASSWORD as string);
+    await this.passwordInputField.fill(password);
     await this.continue({ waitForURL: "**/user/**" });
   }
 
-  async signIn(email: string) {
+  async signIn(email: string, optionalPassword?: string) {
```
Comment (mansaj) [inline src/e2e/pages/authPage.ts:51]: where are we passing in this optional password?

File: src/e2e/specs/dashboard.spec.ts [hunk 3]
```diff
@@ -642,3 +718,42 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Navigation`, (
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Data Breaches`, () => {
+  test.beforeEach(async ({ landingPage, page, authPage }) => {
+    await forceLoginAs(
+      "joe@mailinator.com",
```
Comment (mansaj) [inline src/e2e/specs/dashboard.spec.ts:725]: We usually don't put credentials out in plain text, especially passwords. Especially here you can reuse the var for password.
Even though it's for test cases, it's bad security practices

File: src/e2e/pages/authPage.ts (hunk 2 above)
Comment (mozilloid) [reply src/e2e/pages/authPage.ts:51]: I use that in my forceLogin helper method

**Comment by mozilloid** [comment]
@mansaj I changed the credentials to be part of the environment variables, and used them in the test instead!

**Review by mansaj — APPROVED** [review]

**Pull request merged**